spyder/plugins/ipythonconsole/scripts/spyder-kernels.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Please rename this script to `conda-activate.sh`

Reply:
This name might suggest this is more geberal so it should be more like conda-activate-kernel or something